renderer/wagmiConfig/seedWalletConnector.ts

```
@@ -51,11 +51,14 @@ export function seedWalletConnector(parameters: {
 
     switch (method) {
       case "personal_sign":
-        const signature = await wallet.signMessage(hexToString(params?.[0]));
-        window.seedConnector.showToast({
-          type: "success",
-          message: "Message signed!",
-        });
+        const message = hexToString(params?.[0]);
```

Biome (2.5.1)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)